frontend/benefit/applicant/src/constants.ts

```diff
@@ -185,3 +187,9 @@ export enum VALIDATION_MESSAGE_KEYS {
 }
 
 export const dateRegex = /(?:(?:0?[1-9]|1\d|2\d)\.(?:0?[1-9]|1[0-2])|(?:30\.(?!02)(?:0[1-9]|1[0-2]))|(?:31\.(?:0[13578]|1[02])))\.(?:19|20)\d{2}/;
+export const addressRegex = /^([\d (),./A-Za-zÄÅÖäåö-]+)$/;
+export const companyAccRegex = /^FI\d{16}$/;
+export const phoneRegex = /^((\\+[1-9]{1,4}[ \\-]*)|(\\(\d{2,3}\\)[ \\-]*)|(\d{2,4})[ \\-]*)*?\d{3,4}?[ \\-]*\d{3,4}?$/;
```

Comment:
This doesnt seem to allow country code in the beginning, for example `+358401234567` should be valid (finnish number) but it isn't.
I'm not even sure is it reasonable to try to validate the phonenumber with regex, it's daunting task if we would want to allow country code: https://stackoverflow.com/questions/4338267/validate-phone-number-with-javascript/5059082#5059082

Accepted answer suggests to use specified library like: https://github.com/google/libphonenumber

Reply:
With 3rd party library there should be same solution for python too. https://github.com/google/libphonenumber has some port-libary for python so it should be used in the backend.

Reply:
One more option is to not to validate the phone number (except being non-empty) and count on the users that they will input it correctly. Valid value could be also `0401234567 during week and 0407654321 on weekends` if the number is for human reader. For automatic messages it is not...

Reply:
+358 allowed value will be a a part of next PR, we just discussed it in the refinement, as the backend changed to support that just recently

Reply:
Kesäseteli will use only required as only validation for phonenumber but afaik there is different need for benefit application